plugins/authorized_upload/web_client/templates/authorizeUpload.pug

```diff
@@ -0,0 +1,17 @@
+.g-body-title Authorize upload into folder #[i= folder.name()]
```

Comment:
This UI flow strands the user in this authorize upload view with no way to get back to the folder they were in.

Could this be done as a modal, or else add a "return to folder" or "done" button?

<img width="1242" alt="screen shot 2016-11-13 at 1 49 44 pm" src="https://cloud.githubusercontent.com/assets/595023/20248064/46b4b0e0-a9aa-11e6-83a5-788395fbc533.png">